app/models/transcript.rb

```diff
@@ -23,6 +23,13 @@ class Transcript < ActiveRecord::Base
   has_many :transcript_edits
   has_many :transcript_speakers
 
+  attr_accessor :speakers
```

Comment:
`attr_accessor` defines both a `speakers` (reader) and `speakers=` (writer) method. you won't need the former, since you've defined that two lines down. can probably remove this and define `def speakers=` yourself (as in the comments in the controller)